src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -690,6 +730,15 @@
 short-summary: Manage role assignments.
 """
 
+helps['keyvault role assignment create'] = """
+type: command
+short-summary: Create a new role assignment for a user, group, or service principal.
+examples:
+  - name: Create a role assignment for a specified assignee with a defined role and scope in a Managed HSM using its name. (autogenerated)
+    text: |
+        az keyvault role assignment create --assignee fb2f-ac10--a04f-8b0d786ea37d --hsm-name mock-mhsm-sd --role "Managed HSM Crypto User" --scope "/"
```

Comment:
```suggestion
        az keyvault role assignment create --assignee fb2f-ac10--a04f-8b0d786ea37d --hsm-name mock-mhsm --role "Managed HSM Crypto User" --scope "/"
```
`mock-mhsm-sd` sounds like MHSM security domain file